geomesa-tools/bin/common-functions.sh

```diff
@@ -279,6 +289,57 @@ function registerAutocomplete() {
   fi
 }
 
+function installScala() {
```

Comment:
this function can be removed now i think?

Reply:
Woops, I thought I removed that.